numba/np/arraymath.py

```diff
@@ -1712,7 +1712,7 @@ def np_triu_indices_from_impl(arr, k=0):
     return np_triu_indices_from_impl
 
 
-def _prepare_array(arr):
+def _prepare_array(arr) -> np.ndarray:
```

Comment:
Not sure how this type annotation got in here - my guess is that it was removed some time in the last several months after I originally branched, and then git put it back when I merged master into the feature branch.